pkg/services/ngalert/state/historian/annotation.go

```diff
@@ -40,12 +40,17 @@ func NewAnnotationBackend(annotations annotations.Repository, dashboards dashboa
 }
 
 // RecordStates writes a number of state transitions for a given rule to state history.
-func (h *AnnotationBackend) RecordStatesAsync(ctx context.Context, rule *ngmodels.AlertRule, states []state.StateTransition) {
+func (h *AnnotationBackend) RecordStatesAsync(ctx context.Context, rule *ngmodels.AlertRule, states []state.StateTransition) <-chan error {
```

Comment:
Do we not also need to change something at the call sites of `RecordStatesAsync` for this to do anything? Nothing is ever listening to this channel, we're just returning it into the void.

Reply:
We will do it here https://github.com/grafana/grafana/pull/60734/files#diff-1fe393768892c6c76a43e1ac0ea9ca69ec72d2124d6d80abbede394dd49077aeR521

Reply:
OK, that's fine, you might want to remove `Fixes <issue>` from the description or Github will close the issue prematurely.